community-contributions/joystreamtelegrambot/src/lib/announcements.ts

```diff
@@ -0,0 +1,237 @@
+import { Api, Member, ProposalDetail, Proposals } from "../types";
+import { BlockNumber } from "@polkadot/types/interfaces";
+import { Channel, ElectionStage } from "@joystream/types/augment";
+//import { Channel } from "@joystream/types/channel";
+import { Category, Thread, Post } from "@joystream/types/forum";
+import { domain } from "../../config";
+import {
+  categoryById,
+  memberHandle,
+  memberHandleByAccount,
+  proposalDetail
+} from "./getters";
+
+const sleep = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
+
+const query = async (
+  test: string,
+  callback: () => Promise<any>
+): Promise<any> => {
+  let result: any = await callback();
```

Comment:
Result will implicitly figure out the type to be any here, no need to explicitly write it. 

There's more similar situations in the files.